src/it/MSHADE-316/invoker.properties

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I don't see any Java 8  specific code, in which case this file can be removed.

Reply:
You're right, thanks for the pointer! I have removed that file in ec4c056ac338e4129d72fb81fc51a4b3213ef258.